libraries/integration/Microsoft.Bot.Builder.Integration.NetCore/BotBuilder.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Bot.Builder.Adapters;
+using Microsoft.Bot.Builder.Middleware;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Integration.NetCore
+{
+    internal class BotBuilder : IBotBuilder
```

Comment:
This is an awkward name, given that our SDK is called "BotBuilder". I get the Builder pattern here, but wonder if this isn't more confusion that it's worth...

Reply:
yeah

---
In reply to: [171710856](https://github.com/Microsoft/botbuilder-dotnet/pull/218#discussion_r171710856) [](ancestors = 171710856)

Reply:
... `BotCreator`?

Reply:
It will now be `[I]BotConfigurationBuilder`.